### PR TITLE
Render a loader (if provided) in SSR instead of null

### PR DESCRIPTION
**Pull request by karolis-sh**
https://github.com/gilbarbara/react-inlinesvg/issues/141

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -79,7 +79,6 @@ export default class InlineSVG extends React.PureComponent<IProps, IState> {
     this._isMounted = true;
 
     if (!canUseDOM()) {
-      this.handleError(new InlineSVGError('No DOM'));
```
Comment (gilbarbara) [inline src/index.tsx]: Why did you remove this? If a user didn't set a loader, it should warn him if there's nothing being rendered, right?

File: test/unsupported.spec.tsx [hunk 2]
```diff
@@ -43,9 +46,19 @@ function setup(): Promise<ReactWrapper> {
 }
 describe('unsupported environments', () => {
   it("shouldn't break without DOM ", async () => {
-    const wrapper = await setup();
+    const mockOnLoad = jest.fn();
+
+    const wrapper = mount(
```
Comment (gilbarbara) [inline test/unsupported.spec.tsx]: Why didn't you use the setup here?

**Comment by gilbarbara** [comment]
I think this change should be described in the README

File: src/index.tsx (hunk 1 above)
Comment (karolis-sh) [reply src/index.tsx]: This error implied that `react-inlinesvg` didn't support SSR environments. I don't think onError should be called in the pre-render stage.

File: test/unsupported.spec.tsx (hunk 2 above)
Comment (karolis-sh) [reply test/unsupported.spec.tsx]: As I removed `this.handleError` call from the implementation, the setup Promise would not resolve as neither `onError` or `onLoad` was called.

File: src/index.tsx (hunk 1 above)
Comment (karolis-sh) [reply src/index.tsx]: But is it a breaking change?
SSR is like a setup stage, and the client then calls either `onLoad` or `onError`

Comment (gilbarbara) [reply src/index.tsx]: Yes, if someone was watching this error to change the behavior...

Comment (karolis-sh) [reply src/index.tsx]: Supporting  SSR and throwing errors in it doesn't make much sense don't you think? In theory someone **might** be listening to that specific no DOM error, but as the error is called in `setState` function you can't really adjust the UI in SSR stage as only the first render happens there, at best you could log something to sentry or something like that. We have an SVG wrapper that swaps the output when an error occurs, but that doesn't work in SSRs pre-render stage.

Anyway what do you feel needs to be done here?

Comment (karolis-sh) [reply src/index.tsx]: Hey @gilbarbara , could you hint what you expect from this PR? So I could adjust accordingly.

**Comment by Oikio** [comment]
Any updates on this one?
